raiden/tests/unit/test_pfs_integration_refactored.py

```diff
@@ -67,7 +67,7 @@ def test_get_pfs_info_error():
         with pytest.raises(ServiceRequestFailed) as error:
             get_pfs_info("url")
 
-        assert "Expecting property name enclosed in double quotes:" in str(error.value)
+        assert "Selected Pathfinding service returned unexpected reply" in str(error.value)
```

Comment:
Is the indentation level correct? Shouldn't this assert be part of the second, inner `with` block?

Reply:
I don't think so